script/policy_source_helper.py

```diff
@@ -129,6 +141,27 @@ def AddBravePolicies(template_file_contents):
             'desc': ('''This policy allows an admin to specify that Brave '''
                      '''Shields enabled.'''),
         },
+        {
+            'name': 'BraveVPNDisabled',
+            'type': 'main',
+            'schema': {
+                'type': 'boolean'
+            },
+            'supported_on': ['chrome.*:112-'],
+            'future_on': ['android'],
+            'features': {
+                'dynamic_refresh': False,
+                'per_profile': True,
```

Comment:
Curious why this policy is per-profile?
I think it should be global because reg value is also system-wide value.
and `kManagedBraveVPNDisabled` also should be in local state?
Or is there any reason this should be per-profile?

Reply:
most of above are per profile, policies in the system also can be set per user using HKCU